### PR TITLE
Feature: ND-2 adding structure

### PR DESCRIPTION
**Pull request by HebrineX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
-# pokemon-api-backend
```
Comment (Nicolas-Melluso) [inline README.md]: xD

File: package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@nestjs/common": "9.0.0",
+    "@nestjs/core": "9.0.0",
+    "@nestjs/mongoose": "9.2.1",
+    "@nestjs/platform-express": "9.0.0",
```
Comment (Nicolas-Melluso) [inline package.json:26]: We will use express instead fastify?

File: src/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { UserModule } from './user/user.module';
+import { UserController } from './user/user.controller';
+import { UserService } from './user/user.service';
+import { MuseumController } from './museum/museum.controller';
+import { MuseumService } from './museum/museum.service';
+
+@Module({
+  imports: [PokemonModule, UserModule, MuseumModule],
+  controllers: [PokemonController, UserController, MuseumController],
+  providers: [PokemonService, UserService, MuseumService],
```
Comment (Nicolas-Melluso) [inline src/app.module.ts:16]: Excellent but this works propperly?

**Review by Nicolas-Melluso — CHANGES_REQUESTED** [review]
I don't see any .spec file so even with something to test we need to think in add unit test always

File: package.json (hunk 2 above)
Comment (HebrineX) [reply package.json:26]: Yes! once the structure is "done" migrate to fastify

File: src/app.module.ts (hunk 3 above)
Comment (HebrineX) [reply src/app.module.ts:16]: According to the documentation this should work, when we have a couple more advances we will be able to know if something needs to be corrected

File: README.md (hunk 1 above)
Comment (HebrineX) [reply README.md]: DX

File: package.json (hunk 2 above)
Comment (Nicolas-Melluso) [reply package.json:26]: Okok

File: src/app.module.ts (hunk 3 above)
Comment (Nicolas-Melluso) [reply src/app.module.ts:16]: Excellent, I have no more doubts

**Comment by Nicolas-Melluso** [comment]
@HebrineX resolve the merge conflicts and I'll approbe it to squash and merge it

**Pull request merged**